components/Chat/Chat.styles.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+import { system } from 'ui-kit';
+
+const Chat = {};
+
+const CenteredContent = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Can we alphabetize these properties?

**_Nitpick Ninja_** ™️

Reply:
Ah, yes of course. Old habits die hard!